src/mapeo-project.js

```diff
@@ -93,6 +95,7 @@ export class MapeoProject extends TypedEmitter {
   #loadingConfig
 
   static EMPTY_PROJECT_SETTINGS = EMPTY_PROJECT_SETTINGS
+  #isArchiveDevice
```

Comment:
Nit: could you move this a few lines up, with the rest of the members?

Reply:
Addressed in 2dc69a6a0351cb1db0ff65b813e1b0efdccaccae